{
  "CVE Code": "CVE-2022-1508",
  "Published": "2022-08-31T15:33:01",
  "Updated": "2024-08-03T00:03:06.383Z",
  "Description": "An out-of-bounds read flaw was found in the Linux kernel\u2019s io_uring module in the way a user triggers the io_read() function with some special parameters. This flaw allows a local user to read some memory out of bounds.",
  "ProblemTypes": [
    "CWE-125 - Out-of-bounds Read."
  ],
  "Affected": [
    {
      "vendor": "n/a",
      "product": "Kernel",
      "versions": [
        "Not-Known."
      ]
    }
  ],
  "Reference": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2075533",
    "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=89c2b3b74918200e46699338d7bcc19b1ea12110",
    "https://access.redhat.com/security/cve/CVE-2022-1508",
    "https://ubuntu.com/security/CVE-2022-1508"
  ]
}